doc/mdc-migration.md

```diff
@@ -0,0 +1,54 @@
+# MDC Android migration
+## Status:
+Accepted on 2020-04-16
+
+## Context:
+We have been using app compat design theme from the start, while that worked perfectly it did not allow us to use new material components because those are dependent on `Theme.MaterialComponents.*` app theme. Using new material components allows us to have much more control over component theming like text styles, icon & icon gravity, strokes, corners, etc.,
+
+- Primary reason why we are moving to Material Design Components theme is to use `MaterialButton`,  and avoid using different buttons such as `Button` , `OmegaCenterIconButton` or `PrimarySolidButton`.  The material theme migration will also allows us to use other material components. You can find more components supported by Android and how to theme them [here](https://material.io/components).
```

Comment:
Correction:
"The material theme migration will also **allow** us to use other material components."